### ISSUE TITLE
How to improve this lib

### ISSUE THREAD

**Issue by mathiasrw** [parallel-js/parallel.js#126]
Todo:

- [x] Setup Travis-ci to verify PR's works (in node)
- [x] Merge existing PR that can be merged
- [ ] Lean towards a git-flow attitude by keeping master branch as "last stable version" and adding updates to a new `develop` branch
- [ ] Make sure all tests work in node
- [ ] Make sure all tests work in browsers (see image underneath) 
- [ ] Merge changes from the Parallel2 branch: https://github.com/parallel-js/parallel.js/tree/feature/MaXwellFalstein
- [ ] Get front page examples to work
- [ ] Get lib into CDN
- [ ] Update on npm

[![browser support](https://ci.testling.com/adambom/parallel.js.png)](https://ci.testling.com/adambom/parallel.js)

**Comment by MaXwellFalstein** [comment]
Thank you for the dedicated branch include :)
Would you like any help on the rebirt of parallel.js?
I am happy to work on Git and NPM, when I can.

**Comment by mathiasrw** [comment]
Awesome! Yes please!!!

I think the first thing to be done would be to start merging your work with tests and code from https://github.com/parallel-js/parallel.js/tree/feature/MaXwellFalstein into the library. 

I have a feeling some good git-dance-around would make it possible.

**Comment by mathiasrw** [comment]
@amilajack @MaXwellFalstein 

- The `master` branch should only be updated when a new version has been released

- The `develop` branch is the default branch we use between releases. 

- As we are many people around the table I have locked ´develop` branch for direct push so we all work on our own feature branches and only PR to `develop`. That way we avoid creating too many merge problems with each other while developing. 

- I just found the "project" feature of github. I will split the list in the top comment into issues so we can use the project fuctions...
